client/src/components/QuestionsAndAnswers.jsx

```diff
@@ -1,11 +1,57 @@
 import React from 'react';
 
-const QuestionsAndAnswers = (props) => {
-  return (
-    <div data-testid="QuestionsAndAnswers">
-      <h1>Questions And Answers</h1>
-    </div>
-  );
-};
-
-export default QuestionsAndAnswers;
\ No newline at end of file
+import QandAForm from './QandAForm.jsx';
+import QandAList from './QandAList.jsx';
+import QandAButtons from './QandAButtons.jsx';
+import exampleData from '../../../helpers/QandAExample.js';
+
+class QuestionsAndAnswers extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      questionsAndAnswers: []
+    };
+  }
+
+  static getDerivedStateFromProps(props, state) {
+    let newState = { questionsAndAnswers: [] };
+    newState.questionsAndAnswers = props.questionsAndAnswers;
+    return newState;
+  }
+
+  render() {
+    if (JSON.stringify(this.state.questionsAndAnswers) === '{}') {
```

Comment:
Might cut things down a little to use `!Array.isArray(this.state.questionsAndAnswers)`